specs/slider/slider.md

```diff
@@ -0,0 +1,209 @@
+- `aria-valuemin` -
+- `aria-valuemax` - 
+- `aria-valuenow` - 
+
+*Events*
+- `onValueChange`
+
+*Slots*
+- `backgroundTrack`
+- `foregroundTrack`
+- `sliderTrack`
+- `lowerThumb`
+- `upperThumb`
+
+
+### Anatomy and Appearance
+**Structure:**
+
+```
+  <div
+    role="slider"
+    $aria-valuenow="${x => x.aria-valuenow}"
+    $aria-valuemin="${x => x.aria-valuemin}"
+    $aria-valuemax="${x => x.aria-valuemax}"
+  >
+    <input type="range" 
+      id="${x => x.id}"
+      min="${x => x.min}"
+      max="${x => x.max}"
+      step="${x => x.step}"
+      value="${x => x.value}"
+    >
+    <slot name="label">
+      <div>${x => x.label}</div>
+    </slot>
+    <slot name="backgroundTrack">
+      <div></div>
+    </slot>
+    <slot name="sliderTrack">
+      <div></div>
+    </slot>
+    <slot name="sliderForegroundTrack">
+      <div></div>
+    </slot>
+    <slot name="lowerThumb">
+      <div></div>
+    </slot>
+    <slot name="upperThumb">
+      <div></div>
+    </slot>
+    <slot></slot>
+  </div>
+```
+
+## Implementation
+
+```
+<fast-slider
+    id="slider1"
+    initialValue={32}
+    ariaminvalue={"0"}
+    ariamaxvalue={"40"}
+    ariavaluenow="32"}
+    min={0}
+    max={44}
+    step={1}
+>
+    <h2 slot="label"><b>Density</b></h2>
+    <fast-slider-label
+      label="32"
+      showTickMark="true"
+    >
+    </fast-slider-label>
+</fast-slider>
+```
+
+---
+
+### Accessibility
+
+*Slider* implements the recommended keyboard navigation scheme described [W3C aria-practices](http://w3c.github.io/aria-practices/examples/slider/multithumb-slider.html).
+
+- aria-valuenow will be set as the slider value changes
+- right and up arrow move the slider in increasing value and left and down arrows move it in decreasing increments
+- *Form Input*
+
+
+### Globalization
+
+*Consider whether the component has any special globalization needs such as:*
+
+- *Special RTL handling*
+Slider should render a mirror view in rtl for example:
+![](./images/slider-rtl.png) 
+
+- Strings passed into the component should already be localized
+
+### Test Plan
+
+While testing is still TBD for our web components, I would expect this to align with the testing strategy and not require any additional test support.
+
+---
+
+# Slider label
+
+## Overview
+
+The `slider-label` component provides a default styled label to be placed inside a `slider` component. Users can choose to not show the tickmark as well as provide custom label content.
+
+### Use Cases
+
+- *A designer customizing the component.*
+If they use <slider-label> elements those will be styled a specific way internally and the dev can override any part of that style they want.
+
+### API
+
+*Component Name*
+- `slider-label`
+
+*Attributes*
+- `showTickmark` - boolean to show/hide the tick mark. Default is show.
+- `valuePosition` - used to position the label on the track relative to the min and max values

```

Comment:
Maybe we should simplify these names. `mark` and `position` ?

Reply:
You don't think it should be show-mark? I'm fine with it, just mark seemed non-intuitive to me as a boolean. position makes sense to me